src/server/api/routers/projects.ts

```diff
@@ -249,30 +244,44 @@ export const projectsRouter = createTRPCRouter({
             message: "Round not found",
           });
         try {
-          // Fetch Project applications
-          const projectAttestations: Attestation[] =
-            await fetchApplicationAttestations({
-              round,
-              take: limit,
-              skip: cursor * limit,
-              orderBy: [createOrderBy(orderBy, sortOrder)],
-            });
+          const attestationFetcher = createAttestationFetcher({ round });
 
-          const statusByProjectId: Record<string, ApplicationStatus> =
-            await getApplicationsStatusMapById({
-              round,
-              projectAttestations,
-            });
+          // Fetch application approvals for round by admins
+          const approvals = await attestationFetcher(["approval"], {
```

Comment:
Nice performance improvement of only fetching approval attestations first.

Reply:
thanks for suggesting it!